Client/src/components/Layout/Mobile/index.js

```diff
@@ -7,6 +7,7 @@ import Apply from '../../contanier/student/Apply';
 import Profile from '../../contanier/student/Profile';
 import Login from '../../contanier/Auth';
 import SideBar from '../SideBar';
+import MyCourses from '../../contanier/student/Mycourses'
```

Comment:
```suggestion
import StudentCourses from '../../contanier/student/StudentCourses'
```
it's just suggestion

Reply:
done